From 717fb18b0b4526ffef7ac773b19dc4ff8d71bb9c Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Mon, 26 Jun 2017 09:31:36 +0800
Subject: [PATCH 2020/5242] MLK-15007-4: arm64: dts: imx8qm: enable iommu for
 fec

commit  7b66506b491d71e77f29d20b211e3d24462b7700 from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable iommu for FEC1/2.
The SID is programmed in U-BOOT with value 0x12 that shared
by FEC/FEC2, so FEC1/2 will share one SID and one context.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 7bfd873..96d4687 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -1997,6 +1997,7 @@
 		fsl,num-tx-queues=<3>;
 		fsl,num-rx-queues=<3>;
 		power-domains = <&pd_conn_enet0>;
+		iommus = <&smmu 0x12 0x7f80>;
 		status = "disabled";
 	};
 
@@ -2015,6 +2016,7 @@
 		fsl,num-tx-queues=<3>;
 		fsl,num-rx-queues=<3>;
 		power-domains = <&pd_conn_enet1>;
+		iommus = <&smmu 0x12 0x7f80>;
 		status = "disabled";
 	};
 
-- 
1.7.9.5

